packages/styles/icon-button.css

```
@@ -152,61 +154,83 @@ a.IconButton {
 }
 
 .IconButton--error:active {
-  background-color: var(--icon-button-background-color-error-active);
+  background-color: var(--button-background-color-error-active);
```

This should not be referencing button variables directly, but icon variables:

```suggestion
  background-color: var(--icon-button-background-color-error-active);
```